resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
+      <ComponentRef Id="Service_MARIONETTECOLLECTIVE_svc" />
+    </ComponentGroup>
+
+    <DirectoryRef Id="MARIONETTECOLLECTIVEBINDIR">
+      <Component Id='Service_MARIONETTECOLLECTIVE_svc'
```

Comment:
I mean, I kind of preferred the `MCOService` syntax, since that means we don't have to spell out MARIONETTECOLLECTIVE everytime in capital letters.... also, if you do want to stick with this syntax, can we please please please please use the mco service ID that gets set already?